cluster/common.sh

```diff
@@ -80,7 +80,9 @@ function create-kubeconfig() {
   fi
 
   "${kubectl}" config set-cluster "${CONTEXT}" "${cluster_args[@]}"
-  "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
+  if [ -n "${user_args[@]:-}" ]; then
```

Comment:
These actually evaluate differently. The `[[ -n` does the right thing with array expansion, whereas `[` seems to, well, deal poorly. I love bash:

```console
zml@zml:~$ args=("--foo" "--bar" "--blatz")
zml@zml:~$ if [[ -n "${args[@]:-}" ]]; then echo "WUMPUS"; fi
WUMPUS
zml@zml:~$ if [ -n "${args[@]:-}" ]; then echo "WUMPUS"; fi
-bash: [: too many arguments
```